src/vsTTempSmooth.cpp

```diff
@@ -1,731 +1,1174 @@
 #include "VCL2/instrset.h"
 #include "vsTTempSmooth.h"
 
+// need forceinline ?
+unsigned int INTABS(int x) { return (x < 0) ? -x : x; }
```

Comment:
It was used in _AVX2.cpp as debug scalar way at the debugging of SIMD function. For debug I run both SIMD and scalar functions and compare output and set breakpoint if output is not equal. If keeping this functions for possible future development and debug of SIMD functions in other files is not required - it may be left only in C-functions file.

Reply:
In that case the function definition should be moved to `vsTTempSmooth.h`. Something like:
```
AVS_FORCEINLINE unsigned int INTABS(int x) { return (x < 0) ? -x : x; }
```